satellite/referral/offersweb/server.go

```diff
@@ -4,18 +4,24 @@
 package offersweb
 
 import (
+	"html/template"
 	"context"
 	"net"
 	"net/http"
 	"strings"
+	"runtime"
 
 	"github.com/zeebo/errs"
 	"go.uber.org/zap"
 	"golang.org/x/sync/errgroup"
 )
 
 // Error is satellite referral error type
-var Error = errs.Class("satellite referral error")
+var (
+	Error = errs.Class("satellite referral error")
+	_, fp, _, _ = runtime.Caller(0)
```

Comment:
See this comment for file path https://github.com/storj/storj/pull/1938/files#r282598760

Reply:
You may want to check if the last value as well here

Reply:
@stefanbenten May I ask why ? It's the ok bool value but is there a scenario that it would ever return false? If passing 0 to caller gets the execution stack for the file invoking it then how would it ever be able to return false ?

Reply:
I am fine if there is a valid explanation for it.
Please comment your reason above it for everyone, then youre good! 👍

Reply:
That file path will be bound to your local code location, that may not work for precomputed binaries. Also you set hard rule for path of your static files to be in folder where `server.go` was during compilation. But I am not sure if we care about making static files configurable this much